### DISCUSSION TITLE
UseLenis() hook withour callback

### DISCUSSION THREAD

**Discussion by robinNavas** [darkroomengineering/lenis#316] [Ideas]
Hi, great work on Lenis-scroll !

I think it would make sense to allow the use of useLenis hook without callback function in order to get only the lenis instance. For example I store it as a variable and call lenis.scrollTo() when some condition are fulfilled. 

To be clearer: `const lenis = useLenis()` instead of  `const lenis = useLenis(() => {})`

Maybe I don't use correctly the feature btw.

**Comment by clementroche** [comment]
https://github.com/darkroomengineering/lenis/issues/314
